geometry_msgs/Pose startPose
geometry_msgs/Pose endPose
---
int64 success_bool
